### DISCUSSION TITLE
Ender-3 V3 SE Klipper

### DISCUSSION THREAD

**Discussion by DarthRidar** [CrealityOfficial/Creality_Sonic_Pad#100] [General]
Hello. Does the Ender-3 V3 SE also get Klipper?

**Comment by randybb** [comment]
What do you mean? You can flash the internal MCU with the klipper FW as klipper is totally universal. You can easily create configuration based on its schematics (if you will find it), or just take something from the other people - like this one https://github.com/bootuz-dinamon/ender3-v3-se-full-klipper or https://github.com/0xD34D/ender3-v3-se-klipper-config

But the "official" way is [Creality Sonic Pad ](https://www.creality.com/pages/download-creality-sonic-pad) and it is supported (it is in the menu, so the official template exists) from version 1.0.6.52.92 (Nov 7, 2023) - just google printer-Ender-3V3SE.cfg

**Reply by WolfieSpain** [reply]
So the nebula smart pad is the same as the sonic pad? Even when they are  on the outside compleet deferent as the sonic is with layout horizontal and is klipperscreen the nebula pad is vertical....another problem is no root acces on it after the update .Can you provide username and pasword?

**Comment by DarthRidar** [comment]
OK.  understood.  I have the Ender 3 V3 SE with the Nebula Pad.  I just thought that would work.

**Reply by WolfieSpain** [reply]
yes , i pulled the files from the usa Creality store then Rooted the nebula so i can use orca slicer but when you do the latest update of the nebula you loose the root access so... i hope Creality gives root  access on the nebula pad. Way are they waiting on the community for rooting this device ....

**Comment by randybb** [comment]
Based on the [release notes ](https://www.creality.com/pages/download-creality-nebula-smart-kit)v1.1.0.23 (Dec 11, 2023) should support it

**Reply by WolfieSpain** [reply]
if you do , it fix the problem from the camera but you loose root access i stile trying to figure out if i must re flash the ender3V3SE with a updated version off klipper , but i cant find a newer version for now
